test/metabase/pulse/send_test.clj

```diff
@@ -622,24 +622,6 @@
          (is (mt/received-email-body? :rasta #"Manage your subscriptions"))
          (is (mt/received-email-body? "nonuser@metabase.com" #"Unsubscribe")))))))
 
-(deftest pulse-permissions-test
```

Comment:
this is replaced by `metabase.notification.payload.impl.card-test/permission-test`